### DISCUSSION TITLE
Table isn't accessible after the migration / rollback doesn't work

### DISCUSSION THREAD

**Discussion by onaralili** [supabase/supabase#9620] [Questions] [answered]
I just migrated to the new scheme and it seems my account lost its access to the table.

Cannot access the table via Table Editor (Error: [500] failed to get pg.tables: connect ECONNREFUSED 3.66.215.160:5432).

I tried to rollback and got "Failed to roll back migration: Failed to execute owner reassignment: connect ECONNREFUSED 3.66.215.160:5432" error.

From what I understand, after migration there is a temporary user but it seems it's not possible to grant access to the current user through the dashboard.

Is there way around this?

**Comment by GaryAustin1** [comment]
Assuming you mean https://github.com/supabase/supabase/discussions/9314.   Maybe a team member will see this, but in the meantime they say if you have issues to contact support.

**Reply by onaralili** [reply]
Yep that is the one. I already contacted via the support but their automated email recommended to open an issue here.

**Reply by GaryAustin1** [reply]
Yeah that is boiler plate info.  But maybe someone has encountered it, or a team member sees it here first.

**Reply by onaralili** [reply]
Talking to the support now. They are super helpful :) 
I'm just curious if this removed access to the table is expected behavior once migrated?

**Reply by GaryAustin1** [reply]
Let us know what you find out.

**Reply by onaralili** [reply]
Will do, they offered to roll back for me as the rollback functionality on the dashboard doesn't work. Haven't heard back yet. It seems neither migration nor rollback are stable at this point.

**Comment by onaralili** [comment, marked as answer]
The support team solved the issue manually on their end.
